WordPress/Classes/ViewRelated/Blog/Site Picker/SitePickerViewController.swift

```diff
@@ -88,11 +88,7 @@ extension SitePickerViewController: BlogDetailHeaderViewDelegate {
             showUpdateSiteIconAlert()
         }
 
-        if #available(iOS 14.0, *) {
-            showSiteIconSelectionAlert()
-        } else {
-            showUpdateSiteIconAlert()
-        }
+        showSiteIconSelectionAlert()
```

Comment:
I think this code might possibly show two alerts? if `FeatureFlag.siteIconCreator.enabled == false` it'll call `showUpdateSiteIconAlert()` and `showSiteIconSelectionAlert()`. 

Is that right?

Or, should we add a `return` to the successful feature flag check branch? 

cc @momo-ozawa who's the last one who touched this code. Thanks!

Reply:
That's great! 🙌